presto-spark-launcher/src/main/java/com/facebook/presto/spark/launcher/PrestoSparkLauncherCommand.java

```diff
@@ -52,6 +53,14 @@ public void run()
         TargzBasedPackageSupplier packageSupplier = new TargzBasedPackageSupplier(new File(clientOptions.packagePath));
         packageSupplier.deploy(sparkContext);
 
+        Optional<Map<String, String>> sessionPropertyConfigurationProperties;
+        if (clientOptions.sessionPropertyConfig == null) {
+            sessionPropertyConfigurationProperties = Optional.empty();
+        }
+        else {
+            sessionPropertyConfigurationProperties =
+                    Optional.of(loadProperties(checkFile(new File(clientOptions.sessionPropertyConfig))));
```

Comment:
put into the same line

Reply:
Putting it on one line seems to make this line of code very long.

Reply:
We allow a single line to have 180 charaters.